src/main/java/org/folio/entlinks/controller/AuthorityController.java

```diff
@@ -40,8 +51,13 @@ public ResponseEntity<AuthorityDto> getAuthority(UUID id) {
   }
 
   @Override
-  public ResponseEntity<AuthorityDtoCollection> retrieveAuthorities(Integer offset, Integer limit, String query) {
-    return ResponseEntity.ok(delegate.retrieveAuthorityCollection(offset, limit, query));
+  public ResponseEntity retrieveAuthorities(Boolean deleted, Boolean idOnly, Integer offset, Integer limit,
+                                            String query, @RequestHeader("Content-type") String contentType) {
```

Comment:
Seems that it was wrong description in the story. We should use `Accept` header instead of `Content-type`. `Accept` - is request header and `Content-type` is response header.